Instruments/config/cross_section.cfg

```diff
@@ -75,6 +79,15 @@ VBFHH_bbtautau_cv_1_c2v_1_c3_0: xs="0.004591 * 2 * 5.824e-01 * 6.272e-02" unc="+
 VBFHH_bbtautau_cv_1_c2v_1_c3_2: xs="0.001431 * 2 * 5.824e-01 * 6.272e-02" unc="+0.03-0.04" ref="unknown"
 VBFHH_bbtautau_cv_1_c2v_2_c3_1: xs="0.014392 * 2 * 5.824e-01 * 6.272e-02" unc="+0.03-0.04" ref="unknown"
 VBFHH_bbtautau_cv_1_c2v_0_c3_1: xs="0.027080 * 2 * 5.824e-01 * 6.272e-02" unc="+0.03-0.04" ref="XSDB"
+VBFHH_bbtautau_cv_0.5_c2v_1_c3_1_dipoleRecoilOff: xs="0.010878 * 2 * 5.824e-01 * 6.272e-02" unc="+0.03-0.04" ref="unknown"
```

Comment:
- cross-sections should be always the same, independently whatever diploe Recoil is off or on.
-Please, cross-check that xs values are the same for VBF as suggested for the combination (take 2017&2018 values).